commands/user_e2e_test.go

```diff
@@ -136,3 +138,63 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListUserCmd() {
+	s.SetupTestHelper().InitBasic()
+			
+	// populate map for checking
+	userPool := map[string]bool{
+		s.th.BasicUser.Username : true,
+		s.th.BasicUser2.Username: true,
+		s.th.TeamAdminUser.Username:true,
+		s.th.SystemAdminUser.Username:true,
+	}
+	for i := 0; i < 10; i++ {			
+		usr := s.th.CreateUser()
+		userPool[usr.Username] = true
+	}
+
+	s.RunForAllClients("Get some random of user", func(c client.Client) {
+		printer.Clean()
+
+		page:= int(0)
+		perpage:= int(5)
+		all := bool(false)		
+		cmd := &cobra.Command{}
+		cmd.Flags().IntVar(&page,"page", page,"page")
+		cmd.Flags().IntVar(&perpage,"per-page", perpage,"perpage")
+		cmd.Flags().BoolVar(&all, "all",all,"all")
+
+		err := listUsersCmdF(c, cmd, []string{})
+		s.Require().Nil(err)
+		s.Len(printer.GetLines(), 5)
+		s.Len(printer.GetErrorLines(), 0)
+		log.Println(printer.GetErrorLines())
+
+		for _,each := range printer.GetLines() {
+			eachUser := each.(*model.User)
+			s.Require().Equal(userPool[eachUser.Username], true)
```

Comment:
I'd prefer more convenient variables names, and also we can use `require.Contains` here.
```suggestion
		for _, u := range printer.GetLines() {
			user := u.(*model.User)
			s.Require().Contains(userPool, user.Username)
```

Reply:
sure